src/main/webapp/i18n/de/complaint.json

```diff
@@ -46,6 +42,34 @@
                 "rejected": "Abgelehnt",
                 "showAddressedComplaints": "Adressierte Beschwerden anzeigen",
                 "noComplaintsToShow": "Es gibt im Moment keine Beschwerden zu zeigen."
+            },
+            "review": "Beschwerde überprüfen"
+        },
+        "moreFeedback": {
```

Comment:
Some of the translations in this file are horrible. Please improve them!

Reply:
Can you, please, point where exactly it should be changed?